site/src/pages/AdminPage/Reports/Reports.tsx

```diff
@@ -1,17 +1,40 @@
 import { FC, useCallback, useEffect, useState } from 'react';
 import ReportGroup from './ReportGroup';
+import ReviewItemGrid from '../../../component/ReviewItemGrid/ReviewItemGrid';
 import './Reports.scss';
-import trpc from '../../trpc';
+import trpc from '../../../trpc';
 import { ReportData } from '@peterportal/types';
+import LoadingSpinner from '../../../component/LoadingSpinner/LoadingSpinner';
+
+interface ReviewDisplay {
+  reviewId: number;
+  reports: ReportData[];
+}
+
+const ReportsList: FC<{
+  data: ReviewDisplay[];
+  acceptReports: (reviewId: number) => Promise<void>;
+  denyReports: (reviewId: number) => Promise<void>;
+}> = ({ data, acceptReports, denyReports }) => {
+  return (
+    <ReviewItemGrid>
+      {data.length == 0 && <span>There are currently no reports that need attention</span>}
+      {data.map((reviewPair) => (
+        <ReportGroup
```

Comment:
the name `ReportGroup` was a bit vague until I saw the staging instance but since that's not the concern of this PR i think it's fine